### ISSUE TITLE
BUG: CONTENT_UNAVAILABLE and MEDIA_PLAYBACK: Not Playable

### ISSUE THREAD

**Issue by NicoAiko** [Musish/Musish#426]
**Describe the bug**
Hello, I currently have a Windows PC and I wanted to use Apple Music without having to use iTunes, as this Windows PC is actually my working machine.

I have a Japanese Apple Music account that successfully authenticates but it seems Musish isn't able to play songs, for whatever reason.

**To Reproduce**
1. Login to Musish (with Japanese Account?)
2. Try playing a song
3. Check Dev Tools (console)

**Expected behavior**
Should play the song.

**Screenshots**
![image](https://user-images.githubusercontent.com/22343587/60959206-180bc080-a308-11e9-9661-4c700c9337ee.png)

![image](https://user-images.githubusercontent.com/22343587/60959001-b4819300-a307-11e9-9548-d92199534319.png)

**Desktop (please complete the following information):**
 - Windows 10
 - Chrome
 - v75.0.3770.100

**Additional context**
Also, it seems as my iCloud Music Library songs also aren't playable.

![image](https://user-images.githubusercontent.com/22343587/60959275-34a7f880-a308-11e9-97ae-4d9054d6b94f.png)

**Comment by NicoAiko** [comment]
Extra context: It seems some songs are actually playable.

Japanese/Korean ones seem to not work...

Is there some kind of request Musish is using where the Apple Music API rejects?

**Comment by SpartakusMd** [comment]
Hi @NicoAiko. This seems to be a limitation from Apple. Probably the song is not playable in your region. Could you try playing the song from iPhone/iPad/iTunes? Does it play?

**Comment by NicoAiko** [comment]
Yeah, I can play all my songs without any problems on any of my Apple devices
